VATSpy.dat

```diff
@@ -17790,6 +17790,7 @@ LCCC|Nicosia||LCCC
 LCCC-E|Nicosia|LCCC-E|LCCC-E
 LCCC-S|Nicosia|LCCC-S|LCCC-S
 LCCC-W|Nicosia|LCCC-W|LCCC-W
+LCCC-H|Nicosia-(South2 Sector)|LCCC-H|LCCC-H

```

Comment:
Please move this between `LCCC-E` and `LCCC-S` to keep them alphabetically sorted.